feat: update font v9

gfellerph opened the pull request:
Updates to the new font.

oliverschuerch commented on packages/styles/src/variables/_type.scss:
```diff
@@ -58,9 +58,9 @@ $line-height-sm: 1;
-$font-weight-light: 300 !default; // Design System -> Frutiger Neue Light
-$font-weight-normal: 400 !default; // Design System -> Frutiger Neue Roman
-$font-weight-bold: 700 !default; // Design System -> Frutiger Neue Bold
+$font-weight-light: 300 !default;
+$font-weight-normal: 400 !default;
+$font-weight-bold: 700 !default;
```
What about `$font-weight-black`?
And since we remove the light font variant, we should also remove the variable `$font-weight-light`.

oliverschuerch reviewed (changes requested):
Why do we need to add the new font files in the styles and the docs packages, if we're always loading/using them from the cdn?

gfellerph commented:
> Why do we need to add the new font files in the styles and the docs packages, if we're always loading/using them from the cdn?

That was by mistake, removed them again.

oliverschuerch reviewed (commented):
Why all these changes in the jest files?

gfellerph commented:
> Why all these changes in the jest files?

I removed the forwarded options variable that is being checked in the components tests. This is no longer working, therefore I removed the check in the tests.
